src/components/auth/SignInScreen.js

```diff
@@ -103,104 +103,138 @@ export default class SignInScreen extends React.Component {
                       onChangeText={value => this.onChangeText('password', value)}
                     />
                   </Item>
+                  {/* Sign Up Text */}
+                  <Text style={styles.buttonText2}>Novo usuário?{" "}
+                    <Text 
+                      onPress={() => this.props.navigation.navigate('SignUp')}
+                      style={styles.buttonText2link}>Crie uma conta 
+                    </Text>
+                    {" "}»
```

Comment:
## Try to avoid variable white space.

Reply:
This should use the vector icon instead (from home screen)

Reply:
vector icon is included rather than the carats now. can you elaborate what you mean by variable white space?

Reply:
I think the white space is fine (i've seen george use it too)